code/modules/research/xenoarchaeology/artifact/effects/unknown_effect_gravity.dm

```diff
@@ -0,0 +1,32 @@
+/datum/artifact_effect/gravity
+	effecttype = "gravity"
+	effect_type = 1
+
+	var/pull_strength
+	copy_for_battery = list("pull_strength")
+
+/datum/artifact_effect/gravity/New()
+	..()
+	effectrange = rand(3,12)
+	pull_strength = pick(STAGE_ONE,STAGE_TWO,STAGE_THREE,STAGE_FOUR,10;STAGE_FIVE)
```

Comment:
This makes it 10 times more likely to get STAGE_FIVE than any other stage, right?